### DISCUSSION TITLE
Optional logging of request payload for Audit

### DISCUSSION THREAD

**Discussion by susanta-mukhopadhyay** [9tigerio/db2rest#668] [Ideas]
We may think of logging of request payload for audit purpose simply using distributed file system or object store. I had practical experience on getting this request from various customer's in telecom charging and billings sector.

**Comment by thadguidry** [comment]
@kdhrubo I'd categorize this as an "Enterprise edition only" feature.

**Comment by kdhrubo** [comment]
1. Introduce a parameter called `log_request=true|false`
2. Write to log stream. The logs are written to file system or transferred to wherever .....!

@susanta-mukhopadhyay  - hopefully this helps.
